src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -241,6 +241,12 @@ public override void Execute(DocumentsOperationContext context)
                             cmd.Document.TryGet(Constants.Documents.Metadata.Key, out metadata);
                             LastEtag = putResult.Etag;
 
+                        
+                            metadata.Modifications = new DynamicJsonValue(metadata)
```

Comment:
Yes, but we have Key and Etag already in the reply so why not save the traffic in this hot-path and just take care of this on the client? Now we are sending Metadata back and forth with all other fields.